client/my-sites/github-deployments/use-github-connection-query.ts

```diff
@@ -18,7 +18,7 @@ export interface GithubConnectionData {
 
 export const useGithubConnectionQuery = (
 	siteId: number | null,
-	options?: UseQueryOptions< GitHubConnection[] >
+	options?: Partial< UseQueryOptions< GitHubConnection[] > >
```

Comment:
`Partial` seems sort of suspicious here because the fields in `UseQueryOptions` are already fairly optional. This is another way to fix the issue which I think the `Partial` was added to fix.

```suggestion
	options: Omit< UseQueryOptions< GitHubConnection[] >, 'queryKey' >
```